src/CONST.ts

```diff
@@ -1304,6 +1304,9 @@ const CONST = {
         SIDEBAR_LOADED: 'sidebar_loaded',
         LOAD_SEARCH_OPTIONS: 'load_search_options',
         SEND_MESSAGE: 'send_message',
+        OPEN_SUBMIT_EXPENSE: 'open_submit_expense',
+        OPEN_SUBMIT_EXPENSE_CONTACT: 'open_submit_expense_contact',
+        OPEN_SUBMIT_EXPENSE_APPROVE: 'open_submit_expense_approve',
```

Comment:
@adhorodyski any feedback here on the event names?

Reply:
looking into it